errors/errors.js

```diff
@@ -0,0 +1,21 @@
+exports.handleCustomErrors = (err, req, res, next)=> {
```

Comment:
This won't handle custom errors the way you want it to, it will only fire if there is a 400 which is for a bad request. If there is a bad request this should be handled by your psql error handler NOT a custom error handler